_posts/2019-11-07-tfp-cran/tfp-cran.Rmd

```diff
@@ -0,0 +1,252 @@
+---
+title: "tfprobability 0.8 on CRAN: Now how can you use it?"
+description: > 
+ Part of the r-tensorflow ecosystem, tfprobability is an R wrapper to TensorFlow Probability, the Python probabilistic programming framework developed by Google. We take the occasion of tfprobability's acceptance on CRAN to give a high-level introduction, highlighting interesting use cases and applications.
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2019tfpcran
+date: 11-07-2019
+categories:
+  - Probability and statistics
+  - Packages
+  - Introductions
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/tfprobability.png
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+
+About a week ago, `tfprobability` 0.8 was accepted on CRAN. While on this blog, we've been using this package quite frequently already, in this post we'd like to (re-) introduce it on a high level, especially addressing new users.
+
+## tfprobability, what is it?
+
+`tfprobability` is an R wrapper around [TensorFlow Probability](https://www.tensorflow.org/probability/), a Python library built on top of the [TensorFlow](https://www.tensorflow.org/) framework. So now the question is, what is TensorFlow Probability?
```

Comment:
```suggestion
`tfprobability` is an R wrapper for [TensorFlow Probability](https://www.tensorflow.org/probability/), a Python library built on top of the [TensorFlow](https://www.tensorflow.org/) framework. So now the question is, what is TensorFlow Probability?
```

Reply:
Alternatively, could use "of." "Around" isn't bad—it's the extension of the "wrapping" metaphor—but I think "for" is a bit clearer.